deep_qa/models/multiple_choice_qa/question_answer_similarity.py

```diff
@@ -75,9 +60,15 @@ def _build_model(self):
 
         # Then we pass the question through some hidden (dense) layers.
         hidden_input = regularized_encoded_question
-        for layer in self.hidden_layers:
-            hidden_input = layer(hidden_input)
-        projected_input = self.projection_layer(hidden_input)
+        for i in range(self.num_hidden_layers):
+            hidden_layer = Dense(output_dim=self.hidden_layer_width,
+                                 activation=self.hidden_layer_activation,
+                                 name='question_hidden_layer_%d' % i)
+            hidden_input = hidden_layer(hidden_input)
+        projection_layer = Dense(output_dim=K.int_shape(encoded_answers)[-1],
```

Comment:
It's just a tensor.  You can't call it on a layer object, but it's the same as calling this on an input tensor inside a layer, because this is what shows up in the input.

If I were doing anything more complicated than getting an `int` out of the tensor, and trying to use it somehow here, there would have been problems.